notebooks/community/matching_engine/matching_engine_for_indexing.ipynb

```diff
@@ -261,7 +261,7 @@
         "\n",
```

Comment:
<p>Break text cell into 4 text cells.</p><p>Move Dataset to after Objective.</p><p> </p><p>GCP ANN -&gt; Vertex AI Matching Engine</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Break text cell into 4 text cells - Done</p><p>Move Dataset to after Objective - Done</p><p>GCP ANN -&gt; Vertex AI Matching Engine - Done</p>